pkgs/build-support/build-fhs-userenv/env.nix

```
@@ -124,6 +124,24 @@ let
     paths = [ etcPkg ] ++ basePkgs ++ targetPaths;
     extraOutputsToInstall = [ "out" "lib" "bin" ] ++ extraOutputsToInstall;
     ignoreCollisions = true;
+    postBuild = ''
+      if [ -d  $out/share/gsettings-schemas/ ]; then
```

Prefer bash built-in to `[` command, it does not need quoting the arguments

```suggestion
      if [[ -d  $out/share/gsettings-schemas/ ]]; then
```